sonar-core/src/main/java/org/sonar/core/metric/ScannerMetrics.java

```diff
@@ -74,7 +74,8 @@
 @ScannerSide
 public class ScannerMetrics {
 
-  private static final Set<Metric> ALLOWED_CORE_METRICS = ImmutableSet.<Metric>of(
+  private static final Set<Metric> ALLOWED_CORE_METRICS = ImmutableSet.of(
+    // TODO Lines metric should be removed from this list when we are sure that no existing language plugins is sending measures on it
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)